src/main/java/com/backbase/ct/bbfuel/client/user/UserIntegrationRestClient.java

```diff
@@ -22,41 +20,20 @@
 public class UserIntegrationRestClient extends RestClient {
 
     private final BbFuelConfiguration config;
-
     private static final String SERVICE_VERSION = "v2";
     private static final String ENDPOINT_USERS = "/users";
-    private static final String ENDPOINT_ENTITLEMENTS_ADMIN = ENDPOINT_USERS + "/entitlementsAdmin";
 
     @PostConstruct
     public void init() {
         setBaseUri(config.getDbs().getUser());
         setVersion(SERVICE_VERSION);
     }
 
-    public void ingestEntitlementsAdminUnderLEAndLogResponse(String externalUserId, String externalLegalEntityId) {
-        Response response = ingestEntitlementsAdminUnderLE(externalUserId, externalLegalEntityId);
-
-        if (response.statusCode() == SC_BAD_REQUEST
-            && response.then()
-            .extract()
-            .as(BadRequestException.class)
-            .getMessage()
-            .equals("User is already entitlements admin")) {
-            log.warn("Entitlements admin [{}] already exists under legal entity [{}], skipped ingesting this entitlements admin",
-                    externalUserId, externalLegalEntityId);
-        } else if (response.statusCode() == SC_OK) {
-            log.info("Entitlements admin [{}] ingested under legal entity [{}]",
-                externalUserId, externalLegalEntityId);
-        } else {
-            response.then()
-                .statusCode(SC_OK);
-        }
-    }
-
     public void ingestUserAndLogResponse(UsersPostRequestBody user) {
         Response response = ingestUser(user);
 
-        if (response.statusCode() == SC_BAD_REQUEST &&
+        if (response.statusCode() == SC_BAD_REQUEST
+            &&
```

Comment:
i would not have this on a separate line

Reply:
fixed